### ISSUE TITLE
Dalamud.Boot doesn't like VeraCrypt

### ISSUE THREAD

**Issue by rreminy** [goatcorp/Dalamud#743]
FFXIV is installed inside a VeraCrypt volume and apparently Dalamud.Boot doesn't work properly with this fact. This in turn causes XIVLauncher to display this error and not inject Dalamud on game load *(I been injecting manually for the past few months)*.

![image](https://user-images.githubusercontent.com/27633480/151268440-173b42d2-c369-4700-aac9-3eb472bf6e19.png)

So far I figured out the following:
- Dalamud.Boot would throw `RewriteRemoteEntryPoint failure: CreateFile failure (GetLastError: 3)`, causing the function to return 1 *(non-zero)*.
- This causes XIVLauncher to display the message above. *(Note that I do not have any Anti-Virus installed, including Windows Defender)*
- CreateFile path is `\\?\VeraCryptVolumeR\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\ffxiv_dx11.exe`

Moving it out of the VeraCrypt volume resolves the issue. I do not know if this applies for all types of virtual volumes but would be something take a further look into. This did not use to happen before around `6.0.0.0`.

*(My VeraCrypt version is 1.24 in case that matter)*

**Comment by Soreepeong** [comment]
I remember that I had to convert NT path to DOS path in boot because Reloaded tripped on NT paths when I tried it last time, but if it's working for you by manual injection, then I'll try removing DOS path conversion later. 

On a side note, does it work if you use `Legacy` as injection method instead of `New`?

**Comment by rreminy** [comment]
Yes, it works if I inject manually afterwards after the error message with the game running

No I don't know the `Legacy` and `New` injection methods, I only followed what the code was doing via debug and those were my findings

**Comment by rreminy** [comment]
I found the setting, the Legacy method works fine
